CommonTools/Utils/src/FormulaEvaluator.cc

```diff
@@ -562,6 +562,7 @@ namespace {
   const std::string k_log("log");
   const std::string k_log10("log10");
   const std::string k_TMath__Log("TMath::Log");
+  const std::string k_TMath__Exp("TMath::Exp");
```

Comment:
do we really have to support standard c++ functions also in TMath:: namespace/implementation?

Reply:
The usage of "TMath::Exp" in a JEC expression initially triggered the issue, but then plain C++ "exp" was used, so happy with dropping it in order to not unnecessarily clutter the list of expressions (TMath::Exp just uses standard exp anyway --> https://root.cern.ch/doc/master/TMath_8h_source.html#l00726).